web-registry/src/config/errors.tsx

```diff
@@ -0,0 +1,36 @@
+import { BrokenLinkIcon } from 'web-components/lib/components/icons/BrokenLinkIcon';
+import SellOrderNotFoundIcon from 'web-components/lib/components/icons/SellOrderNotFoundIcon';
+
+export enum ERRORS {
+  DEFAULT,
+  SELL_ORDER_PURCHASED,
+}
+
+export const errorsMapping: Record<
```

Comment:
Basic error system that we can extend and use across the app.
The idea is to use the code field later on to match the ledger errors.